test/test_pg_client_store.js

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import db  from '../src/pg_store';
+import should from 'should';
+
+describe('db for api clients', () => {
+  it('1. should succeed to clear db', done => {
+    db.clear().then(done);
+  });
+
+  it('2. should be an empty db', done => {
+    db.getAll().then(
+      (res) => {
+        res.length.should.be.equal(0);
+        done();
+      },
+      () => {
+        should.ok(false);
+      }
+    );
+  });
+
+  it('3. should not have the `test` name', done => {
+    db.hasName('test')
+      .then(
+        () => { should.ok(false); },
+        () => { done(); }
+      );
+  });
+
+  it('4. should add a test client', done => {
+    const client = { name: 'test', key: 'key1', secret: 'secret1' };
+    db.add(client).then(
+      (res) => {
+        res.name.should.be.equal('test');
+        res.key.should.be.equal('key1');
+        res.secret.should.be.equal('secret1');
```

Comment:
Use `client.name`, `client.key` and `client.secret` instead of literals, please.